Marlin/Configuration_adv.h

```diff
@@ -820,20 +820,22 @@
   //#define OVERLAY_GFX_REVERSE
 
   /**
-  * ST7920-based LCDs can emulate a 16 x 4 character display using
-  * the ST7920 character-generator for very fast screen updates.
-  * Enable LIGHTWEIGHT_UI to use this special display mode.
-  *
-  * Since LIGHTWEIGHT_UI has limited space, the position and status
-  * message occupy the same line. Set STATUS_EXPIRE_SECONDS to the
-  * length of time to display the status message before clearing.
-  *
-  * Set STATUS_EXPIRE_SECONDS to zero to never clear the status.
-  * This will prevent position updates from being displayed.
-  */
-  //#define LIGHTWEIGHT_UI
-  #if ENABLED(LIGHTWEIGHT_UI)
-    #define STATUS_EXPIRE_SECONDS 20
+   * ST7920-based LCDs can emulate a 16 x 4 character display using
+   * the ST7920 character-generator for very fast screen updates.
+   * Enable LIGHTWEIGHT_UI to use this special display mode.
+   *
+   * Since LIGHTWEIGHT_UI has limited space, the position and status
+   * message occupy the same line. Set STATUS_EXPIRE_SECONDS to the
+   * length of time to display the status message before clearing.
+   *
+   * Set STATUS_EXPIRE_SECONDS to zero to never clear the status.
+   * This will prevent position updates from being displayed.
+   */
+  #if ENABLED(U8GLIB_ST7920)
```

Comment:
This defeats the purpose of the PR.  If a live human enables LIGHTWEIGHT_UI and observe nothing different and no warning, they are left to scratch their head wondering what they did wrong, unless they are intimately familiar with the display used in their LCD/Control Panel